src/main/kotlin/org/axonframework/intellij/ide/plugin/util/PSiProcessingUtils.kt

```diff
@@ -47,15 +47,18 @@ import org.jetbrains.uast.toUElement
 
 /**
  * Convenience method to fully qualified name of type.
- * Throws if we get a type we do not expect so we can support it.
  */
 fun PsiType?.toQualifiedName(): String? = this?.let {
-    return when (this) {
-        is PsiClassReferenceType -> this.resolve()?.qualifiedName
-        // Class<SomeClass> object. Extract the <SomeClass> and call this method recursively to resolve it
-        is PsiImmediateClassType -> (this.parameters.firstOrNull() as PsiClassType?)?.toQualifiedName()
-        is PsiWildcardType -> "java.lang.Object"
-        else -> null
+    return try {
```

Comment:
Is it at all doable to create a test case for this?

Reply:
I will try, the problem is that I don't have the original code that caused the error, I only see what effect is caused. By removing the useless cast it can't happen again

Reply:
This is also the reason I wrapped it with a try/catch. The exception message will contain the code if it happens again.

Reply:
After some experimentation, the following code causes this issue:

```
import java.util.List;

@AggregateRoot
class MyAggregate {
   @AggregateMember
    private List<?> entities;
}
```

Testcase coming up